Changelog.md

```diff
@@ -23,6 +23,7 @@ Language Features:
 Compiler Features:
  * EVM: Support for the EVM Version "Cancun".
  * SMTChecker: Support `bytes.concat` except when string literals are passed as arguments.
+ * SMTChecker: Print a message that function parameter name was used instead of a concrete value when counterexample generated by a solver is too long.
```

Comment:
```suggestion
 * SMTChecker: Print a message that function parameter name was used instead of a concrete value in a counterexample when the concrete value found by the solver is too long to print.
```